src/node-file-trace.js

```diff
@@ -71,6 +72,7 @@ class Job {
       }
     }
     this.base = base;
+    this.cwd = resolve(processCwd || base);
```

Comment:
Where is `this.cwd` used? I don't see it 🤔

Reply:
I only see `job.cwd` above but that cant be the only change right?

Reply:
`this.cwd` is `job.cwd`

Reply:
Indeed. I was expecting more changes than this one line 😅 

Something is still wrong though as I mentioned with the Next.js example in slack.